Assets/Scripts/Models/Area/World.cs

```diff
@@ -51,6 +51,11 @@ public World(int width, int height, int depth)
         // Creates an empty world.
         SetupWorld(width, height, depth);
         int seed = UnityEngine.Random.Range(0, int.MaxValue);
```

Comment:
it takes an int, and the docs don't *say* anything about limiting the input range, so I *assume* negative is ok.